packages/react/src/lib/factories.ts

```diff
@@ -135,8 +118,8 @@ function createShorthandFromValue({
   options,
 }: {
   Component: React.ReactType
-  mappedProp?: string
-  mappedArrayProp?: string
+  mappedProp?: string | number | symbol
```

Comment:
Why it can be `number` or `symbol`?

Reply:
@layershifter this is WIP - please, do not review yet.

For the question you've asked - this was necessary to match indexer definitions for the typing approach previously used, when it was simplified this has been removed.

But, once again, please, refrain from review now, because there will be some changes in addition, so these comments may be not relevant